src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelAPITests.cs

```diff
@@ -4156,6 +4156,147 @@ void M(string S1 = $""Testing"", Namae n = null)
             Assert.Equal("Level 5", model.GetConstantValue(actual[7]).Value);
         }
 
+        [WorkItem(976, "https://github.com/dotnet/roslyn/issues/976")]
+        [Fact]
+        public void ConstantValueOfRawInterpolatedString()
+        {
+            var source = @"
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine($""""""Hello, world!"""""");
+        Console.WriteLine($""""""{DateTime.Now.ToString()}.{args[0]}"""""");
+    }
+}";
+
+            var comp = CreateCompilation(source, options: TestOptions.ReleaseExe);
+            var tree = comp.SyntaxTrees.Single();
+            var model = comp.GetSemanticModel(tree);
+
+            var actual = tree.GetRoot().DescendantNodes().OfType<InterpolatedStringExpressionSyntax>().ToArray();
+            Assert.True(model.GetConstantValue(actual[0]).HasValue);
+            Assert.Equal("Hello, world!", model.GetConstantValue(actual[0]).Value);
+            Assert.Equal(SpecialType.System_String, model.GetTypeInfo(actual[0]).Type.SpecialType);
+            Assert.False(model.GetConstantValue(actual[1]).HasValue);
```

Comment:
Do we have a test for GetTypeInfo on the expression in interpolation hole?

Reply:
yup.  it's in teh tests below.